package.json

```diff
@@ -26,6 +26,7 @@
   ],
   "homepage": "https://github.com/amilajack/eslint-plugin-compat#readme",
   "scripts": {
+    "benchmarks": "cross-env NODE_ENV=production babel test/benchmarks.ts --out-dir benchmarks-tmp --source-maps inline --extensions '.ts' && node -r source-map-support/register benchmarks-tmp/benchmarks.js",
```

Comment:
Sorry for missing this in the earlier review. We could use [`@babel/node`](https://babeljs.io/docs/en/next/babel-node.html) to simplify this

Reply:
Is it worth even with this caveat?

https://babeljs.io/docs/en/next/babel-node.html#not-meant-for-production-use